src/components/notes/editor-view.jsx

```diff
@@ -42,6 +43,23 @@ function EditorView(props) {
     [onTitleChange]
   )
 
+  useMemo(() => collabProvider && collabProvider.setReadOnly(readOnly), [
+    readOnly,
+    collabProvider
+  ])
```

Comment:
I find it odd to use `useMemo` but to discard the memoized value. I guess the goal is to avoid calling `collabProvider.setReadOnly(readOnly)` when `readOnly` hasn't changed?

Reply:
yep

Reply:
> Rappelez-vous que la fonction fournie à useMemo s’exécute pendant le rendu. N’y faites rien que vous ne feriez pas normalement pendant un rendu. Par exemple, les effets de bord doivent passer par useEffect, et non useMemo.

At first I was going to comment that useEffect could be more correct then I re-read the docs and found that the useMemo creator function was executed during render, so I guess this is normal and wanted to useMemo. I think it is not very clear though, can you add a comment ?